tests/single.rs

```diff
@@ -50,23 +54,16 @@ describe! thunk {
         it "should drop internal data just once" {
             let counter = Arc::new(Mutex::new(0));
             let counter_clone = counter.clone();
-            let result =  Thread::scoped(move || {
+            thread::scoped(move || {
                 let value = Dropper(counter_clone);
                 let t = Thunk::<()>::new(move || {
                     // Get a reference so value is captured.
                     let _x = &value;
-
-                    panic!("Muahahahah")
```

Comment:
It's no longer possible to handle panics from scoped threads ([PR #22435](https://github.com/rust-lang/rust/pull/22435)). I'm not entirely sure if the panic was relevant to the test logic.